zerver/openapi/zulip.yaml

```diff
@@ -8906,6 +8906,23 @@ paths:
               - 2
               - 3
           example: 1
+        - name: show_unread_counts
+          in: query
+          description: |
+            This is used to configure in which streams does user wants to see unread messages count in left sidebar.
```

Comment:
This could use a bit of polishing; "Configuration for which streams should be displayed with a numeric unread count in the left sidebar. Streams that do not have an unread count will have a simple dot indicator for whether there are any unreads messages."